connectors/src/connectors/zendesk/lib/node-zendesk-types.d.ts

```diff
@@ -118,34 +119,39 @@ declare module "node-zendesk" {
     brand: {
       list: () => Promise<{
         response: Response;
-        result: Brand[];
+        result: ZendeskFetchedBrand[];
+      }>;
+      show: (brandId: number) => Promise<{
+        response: Response;
+        result: { brand: ZendeskFetchedBrand };
       }>;
-      show: (
-        brandId: number
-      ) => Promise<{ response: Response; result: { brand: Brand } }>;
     };
     helpcenter: {
       categories: {
-        list: () => Promise<Category[]>;
+        list: () => Promise<ZendeskFetchedCategory[]>;
         show: (
           categoryId: number
-        ) => Promise<{ response: Response; result: Category }>;
+        ) => Promise<{ response: Response; result: ZendeskFetchedCategory }>;
       };
       articles: {
-        list: () => Promise<Article[]>;
+        list: () => Promise<ZendeskFetchedArticle[]>;
         show: (
           articleId: number
-        ) => Promise<{ response: Response; result: Article }>;
-        listByCategory: (categoryId: number) => Promise<Article[]>;
-        listSinceStartTime: (startTime: number) => Promise<Article[]>;
-      };
-      tickets: {
-        list: () => Promise<Ticket[]>;
-        show: (
-          ticketId: number
-        ) => Promise<{ response: Response; result: Ticket }>;
+        ) => Promise<{ response: Response; result: ZendeskFetchedArticle }>;
+        listByCategory: (
+          categoryId: number
+        ) => Promise<ZendeskFetchedArticle[]>;
+        listSinceStartTime: (
+          startTime: number
+        ) => Promise<ZendeskFetchedArticle[]>;
       };
     };
+    tickets: {
```

Comment:
note: this is a fix and not a change, the tickets are definitely not part of the help center